### DISCUSSION TITLE
Setup development environment on Windows

### DISCUSSION THREAD

**Discussion by Elgcahlxukuth** [Try/OpenGothic#288] [Q&A] [answered]
Hi,
Could comeone please advise how to setup dev. env. for OpenGothic project on Windows?
Do you use Visual Studio? or something else?
Best regards

**Comment by Try** [comment, marked as answer]
Hi @Elgcahlxukuth !

I'm using QtCreator as my main IDE and VisualStudio as secondary(for CPU profiling). Practically any CMake-friendly ide should be good to go.
Other than that: WindowsSDK(including DX12) + VulkanSDK.
Also: don't forget to close submodules as well. 

This should be it, unless I forgot some minor stuff.
